.github/workflows/dart_frog.yaml

```diff
@@ -21,3 +21,21 @@ jobs:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/dart_package.yml@v1
     with:
       working_directory: packages/dart_frog
+
+  pana:
+    defaults:
+      run:
+        working-directory: packages/dart_frog
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: dart-lang/setup-dart@v1
+
+      - name: Install Dependencies
+        run: |
+          dart pub get
+          dart pub global activate pana
+      - name: Verify Pub Score
+        run: ../../tool/verify_pub_score.sh
```

Comment:
Right now it holds a single file, but in the context of naming things shouldn't it be plural? Tools?

Reply:
I was going off of the naming convention in other dart projects haha https://github.com/dart-lang/pub/tree/400f21e9883ce6555b66d3ef82f0b732ba9b9fc8/tool

Reply:
Happy to adjust it though 👍

Reply:
But in our naming conventions we prefer plurals right?🤔